assets/keymaps/vim.json

```diff
@@ -672,7 +672,7 @@
     }
   },
   {
-    "context": "EmptyPane || SharedScreen",
+    "context": "EmptyPane || SharedScreen || Workspace",
```

Comment:
Huh. This breaks the tests because `:%s/foo/bar` doesn't work anymore. Any ideas @ConradIrwin? I would've assumed that Workspace has lower precedence than editor, which is where `:%s/foo/bar` should work, right?

Reply:
The test is failing because you can no longer type a `:` into the search bar, because that is in the workspace too.

I'm not sure what the best way of solving this is, but I think adding a DiagnosticsView here is probably the right choice.

I've wanted to do something like "Workspace > !Editor", but that requires a much larger change to how contexts are evaluated, and I'm not sure that we want to make contexts even more complicated :D.